packages/components/src/components/Chip/Chip.module.css

```diff
@@ -0,0 +1,25 @@
+.container {
+  --chip-container-gap: var(--sizes-dds-spacing-x-0-25);
+  --chip-container-border: 1px solid var(--dds-color-neutrals-gray-3);
+  --chip-container-border-radius: var(--radii-dds-border-radius-1-radius);
+  --chip-container-background-color: var(--dds-color-neutrals-gray-1);
+  --chip-container-padding: var(--sizes-dds-spacing-x-0-125)
+    var(--sizes-dds-spacing-x-0-25) var(--sizes-dds-spacing-x-0-125)
+    var(--sizes-dds-spacing-x-0-5);
```

Comment:
Lagde dette som et forslag. Usikker på om jeg liker det 🙈 Prøver å etterligne `Chip.tokens.tsx`, men det gir kanskje ikke veldig mye mening i denne konteksten.

Alternativet er å bruke dds-tokens direkte i CSS under. Hva tenker du?

Reply:
Ikke sant, det er litt utfordrende 😄 Vi skal endre tokens på et tidspunkt til å være ordentlige semantiske tokens, og det kan hende vi skal også ha et lag på komponentnivå på samme måte som dette. Tenker det er ok å bare bruke dds-tokens direkte for nå 👍

Reply:
Tror det gir mer mening på dette tidspunktet å bruke de direkte ja. Endrer til det! 😊